client/hijack.go

```diff
@@ -188,15 +188,25 @@ func (cli *Client) setupHijackConn(req *http.Request, proto string) (net.Conn, e
 
 	c, br := clientconn.Hijack()
 	if br.Buffered() > 0 {
-		// If there is buffered content, wrap the connection
-		c = &hijackedConn{c, br}
+		// If there is buffered content, wrap the connection.  We return an
+		// object that implements CloseWrite iff the underlying connection
```

Comment:
Minor nit, s/iff/if/.

Reply:
I actually meant iff, as in "if and only if" :)